lib/resty/openidc.lua

```diff
@@ -949,8 +936,13 @@ local function is_algorithm_expected(jwt_header, expected_algs)
     return true
   end
   if type(expected_algs) == 'string' then
-    expected_algs = { expected_algs }
+    if expected_algs == jwt_header.alg then
+        return true
+    else
+        return false
+    end
```

Comment:
this could even be simplified to `return expected_algs == jwt_header.alg`

Reply:
Done